File: Java/1282.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int k = 0;
		
		for(int i = n; i > 0; i--){
			double t = Math.sqrt((double)i);
			
			if(t - (int)t == 0){
				System.out.printf("%d %.0f\n", k, t);
				break;
			} else{
				k++;
			}
		}
	}
}

File: C/1282.c
#include <stdio.h>
#include <math.h>

int main(){
	int n, k = 0;
	scanf("%d", &n);
	
	for(int i = n; i > 0; i--){
		double t = sqrt((double)i);
		
		if(t - (int)t == 0){
			printf("%d %.0f\n", k, t);
			break;
		} else{
			k++;
		}
	}
	return 0;
}